app/registries/foreman/plugin.rb

```diff
@@ -79,6 +79,8 @@ def register(id, &block)
         plugin.after_initialize
 
         registered_plugins[id] = plugin
+      rescue ActiveRecord::NoDatabaseError, ActiveRecord::StatementInvalid => e
```

Comment:
Looks like rubocop complaints:

```
app/registries/foreman/plugin.rb:82:7: W: Lint/ShadowedException: Do not shadow rescued Exceptions.
      rescue ActiveRecord::NoDatabaseError, ActiveRecord::StatementInvalid => e ...
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
actually, `ActiveRecord::NoDatabaseError` is redundant as it inherits from `ActiveRecord::StatementInvalid`

https://github.com/rails/rails/blob/09ff877610a09d930ca84a94113f678d4e79e100/activerecord/lib/active_record/errors.rb#L186-L188